lib/utils.js

```diff
@@ -266,11 +272,14 @@ module.exports.getModuleChunk = getModuleChunk;
  * @param {Object} options
  * @param {string} options.chunkName
  * @param {string} options.context
+ * @param {string} options.fullHash
```

Comment:
this is webpack's `fullHash` though, happy to rename it but I figured I would keep it consistent